vcap/vcap/capsule.py

```diff
@@ -171,7 +159,14 @@ def default_options(self):
     # These things are to be filled in by a subclassing capsule
     @property
     def name(self) -> str:
-        """A name to uniquely identify the capsule."""
+        """The name of the capsule. This value uniquely defines the capsule and
+        cannot be shared by other capsules.
+
+        By convention, the capsule's name is prefixed by some short description
+        of the role it plays ("detector", "recognizer", etc) followed by the
+        kind of data it relates to ("person", "face", etc) and, if necessary,
+        some differentiating factor ("fast", "close_up", "retail", etc).
```

Comment:
I would suggest adding an example of a capsule name

Reply:
Dun it!